spec/mongo/auth/scram256/conversation_spec.rb

```diff
@@ -0,0 +1,198 @@
+require 'spec_helper'
```

Comment:
This was copied from spec/mongo/auth/scram/conversation_spec.rb and modified as necessary.